File: FrontEnd/MshopUI/src/app/models/outward/outward.ts
import { BaseModel } from '../base-model/base-model';

export class Outward extends BaseModel{
        OutwardID: string = "";
        OutwardNo: string = "";
        OutwardDate: Date = new Date();
        ShopID: string ="";
        TotalMoney: number;
        OutStaffName: string = "";
        OutStaffCode: string ="";
        InStaffName: string ="";
        InStaffCode: string ="";
        Deliver: string ="";
        Description: string="";
        IsSelected?: boolean = false;
}

File: BackEnd/Models/Dictionary/Outward/Outward.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Models
{
    public class Outward : BaseModel
    {
        public Guid OutwardID { get; set; }
        public string OutwardNo { get; set; }
        public DateTime OutwardDate { get; set; }
        public Guid ShopID { get; set; }
        public decimal TotalMoney { get; set; }
        public string OutStaffName { get; set; }
        public string OutStaffCode { get; set; }
        public string InStaffName { get; set; }
        public string InStaffCode { get; set; }
        public string Deliver { get; set; }
        public string Description { get; set; }
    }
}
